### PR TITLE
Adding supp info

### PR DESCRIPTION
**Pull request by godmodegalactus**

### CODE REVIEW

File: migration.sql [hunk 1]
```diff
@@ -10,20 +10,22 @@ CREATE TABLE banking_stage_results.transaction_infos (
...
-  processed_slot BIGINT
+  processed_slot BIGINT,
+  supp_infos text
 );
 
 CREATE TABLE banking_stage_results.blocks (
-  block_hash CHAR(44) PRIMARY KEY,
-  slot BIGINT,
+  block_hash CHAR(44),
+  slot BIGINT PRIMARY KEY,
```
Comment (grooviegermanikus) [inline migration.sql:19]: nit: move the primary key column to top

File: src/postgres.rs [hunk 2]
```diff
@@ -83,7 +84,7 @@ impl PostgresSession {
         Ok(client)
     }
 
-    pub fn multiline_query(query: &mut String, args: usize, rows: usize, types: &[&str]) {
+    pub fn _multiline_query(query: &mut String, args: usize, rows: usize, types: &[&str]) {
```
Comment (grooviegermanikus) [inline src/postgres.rs:87]: why did you prefix with underscore? "_"

File: src/postgres.rs [hunk 3]
```diff
@@ -117,10 +118,35 @@ impl PostgresSession {
...
+                ) FROM STDIN BINARY
+            "#,
+        );
+        let sink: CopyInSink<bytes::Bytes> = self.copy_in(&statement).await.unwrap();
+        let writer = BinaryCopyInWriter::new(
+            sink,
+            &[
+                Type::TEXT,
+                Type::CHAR,
+                Type::CHAR,
```
Comment (grooviegermanikus) [inline src/postgres.rs:137]: pls check the types - this is for `is_executed` which is a bool

File: src/postgres.rs [hunk 4]
```diff
@@ -117,10 +118,35 @@ impl PostgresSession {
...
+                Type::INT8,
+                Type::TIMESTAMPTZ,
+                Type::TEXT,
+                Type::INT8,
+                Type::TEXT,
+            ],
+        );
+        pin_mut!(writer);
         for tx in txs.iter() {
+            let mut args: Vec<&(dyn ToSql + Sync)> = Vec::with_capacity(NUMBER_OF_ARGS);
```
Comment (grooviegermanikus) [inline src/postgres.rs:149]: I would create the slice right away without Vec/push

File: src/main.rs [hunk 5]
```diff
@@ -197,7 +159,85 @@ async fn main() {
...
+
+#[tokio::main()]
+async fn main() {
+    tracing_subscriber::fmt::init();
+
+    let args = Args::parse();
+
+    let _prometheus_jh = PrometheusSync::sync(args.prometheus_addr.clone());
+
+    println!("x token is {}", args.grpc_x_token.clone().unwrap_or_default());
```
[the remark this answers is not in the document]
Comment (godmodegalactus) [reply src/main.rs:175]: ok

File: src/postgres.rs (hunk 4 above)
Comment (godmodegalactus) [reply src/postgres.rs:149]: The code exisits lazy to remove the vector and sloce

File: src/postgres.rs (hunk 3 above)
Comment (godmodegalactus) [reply src/postgres.rs]: done

**Review by grooviegermanikus — APPROVED** [review]
LGTM

**Pull request merged**